### ISSUE TITLE
codepen.io / skypack.dev usage?

### ISSUE THREAD

**Issue by bumi** [dolcalmi/lnurl-pay#3]
I was trying to create some lnurl-pay demos in [codepen](https://codepen.io). To load the package there codepen seems to use [skypack.dev](https://www.skypack.dev/). Loading lnurl-pay there seems to have a build error: 
[https://cdn.skypack.dev/lnurl-pay](https://cdn.skypack.dev/lnurl-pay)

This is probably not the right place for this and probably not a lnurl-pay issue, but do you have an idea what could be the problem there?
Thanks.

**Comment by bumi** [comment]
hmm, I think this is likely an issue for in-browser usage? and maybe related to the crypto function for the sha256.

**Comment by dolcalmi** [comment]
yep, I will try to fix it this week. Galoy mobile devs also reported this. 

probably related to this config https://github.com/dolcalmi/lnurl-pay/blob/main/config/webpack.config.js#L29

thanks

**Comment by bumi** [comment]
yes, sounds like that could be it. The browser needs some fallback or alternative there. 
I failed to fix it so far :/

**Comment by johnsBeharry** [comment]
any progress on this @dolcalmi ?

**Comment by dolcalmi** [comment]
should be fixed in v0.2.3... 

Example (you can build bundle with `yarn build` or `yarn build:umd`)
```js
<script type="text/javascript" src="dist/umd/index.js" /> 
```
```js
LnurlPay.requestInvoice({
  lnUrlOrAddress: 'user@domain.co',
  tokens: 333,
})
.then(i => console.log(i))
.catch(e => console.error(e))
```
